src/lib/InstantSearch.js

```diff
@@ -227,6 +227,18 @@ To help you migrate, please refer to the migration guide: https://community.algo
 
       this.helper.search();
     }
+
+    if (process.env.NODE_ENV === 'development' && !this.started) {
+      clearTimeout(this._widgetTimer);
+      this._widgetTimer = setTimeout(
+        () =>
+          // eslint-disable-next-line no-console
+          console.warn(
```

Comment:
NOTE: in v3 this will need to be changed to the logger function we added for consistency

Reply:
It's already available in v2: https://github.com/algolia/instantsearch.js/blob/develop/src/lib/utils.js#L426

@HeroProtagonist You can replace `console.warn` with this utility function :)

Reply:
@francoischalifour 

In the constructor, `console.warn` is called a few times (I copied log format from there). Would you want me to add the utility function in those places as well?
https://github.com/algolia/instantsearch.js/blob/develop/src/lib/InstantSearch.js#L137

Reply:
Sure, feel free to update those in another PR!